qiime2/core/archive/provenance.py

```diff
@@ -661,3 +662,19 @@ def fork(self, name, alias):
         forked.alias = alias
         forked.add_ancestor(alias)
         return forked
+
+
+class ReportProvenanceCapture(ProvenanceCapture):
+    def __init__(self):
+        super().__init__()
+        self.inputs = OrderedKeyValue()
```

Comment:
What do we get from using this OrderedKeyValue class? Especially since dicts are fundamentally ordered now

Reply:
I think it was for a specific YAML representation, we connect a custom type (list of single key-value pairs)